src/utils.ts

```diff
@@ -29,3 +32,22 @@ export function epochToBatchId(epoch: BigInt): BigInt {
 export function getBatchId(event: EthereumEvent): BigInt {
   return epochToBatchId(event.block.timestamp)
 }
+
+export function getOwlDecimals(): BigInt {
+  return OWL_DECIMALS
+}
+
+export function coalesce<T>(valueOpt: T | null, defaultValue: T): T {
+  if (valueOpt == null) {
+    return defaultValue
+  } else {
+    return valueOpt!
+  }
+}
+
+export function toPriceInUnits(price: BigInt, decimalsBaseOpt: BigInt | null, decimalsQuoteOpt: BigInt | null): BigInt {
+  let decimalsBase = coalesce<BigInt>(decimalsBaseOpt, DEFAULT_DECIMALS).toI32()
+  let decimalsQuote = coalesce<BigInt>(decimalsQuoteOpt, DEFAULT_DECIMALS).toI32()
+
+  return price.div(TEN.pow(u8(decimalsBase - decimalsQuote)))
```

Comment:
It was one of my fights. Although it says it needs a BigInt, the compiler requires an unisigned 8 bytes var.

This is how you do casting of i32 to u8, since I know the decimals are small, there's no problem with that